misc/SWC_badge/badge_snippet.html

```diff
@@ -0,0 +1,4 @@
+<html>
+<body>
+<a href="http://software-carpentry.org"><img src="file:///D:/Documents/public_github_repos/communications/misc/SWC_badge/SWC_badge.png"></a>
```

Comment:
I think this link is pointing to your local copy of the bagde. Shouldn't be something like https://github.com/swcarpentry/communications/blob/master/misc/SWC_badge/SWC_badge.png instead?